lib/spree/shipment_notice.rb

```diff
@@ -29,6 +44,7 @@ def update
       shipment.update_attribute(:tracking, tracking)
 
       unless shipment.shipped?
+        shipment.reload.ready! unless shipment.ready?
```

Comment:
Mmm yeah if it gets paid it should transition to ready status you are right I will take this out